package.json

```
@@ -31,5 +31,7 @@
     "prettier": "^2.0.5",
     "typescript": "^3.9.6"
   },
-  "dependencies": {}
+  "dependencies": {
+    "react-intl": "^7.1.6"
+  }
```

```suggestion
  "dependencies": {}
```

Not necessary this dependency outside react folder